src/database/migrations/20200927093745-create-user.js

```diff
@@ -7,6 +7,16 @@ module.exports = {
         primaryKey: true,
         type: Sequelize.INTEGER
       },
+      roleId: {
+        allowNull: true,
+        type: Sequelize.INTEGER,
+        onDelete: 'CASCADE',
+        onUpdate: 'CASCADE'
+      },
+      managerId: {
+        allowNull: true,
+        type: Sequelize.INTEGER
+      },
```

Comment:
Remember do not edit old migrations, create a new one.

Reply:
Done